explorer/src/components/Leaderboard/NominatorRewardsList.tsx

```diff
@@ -89,6 +89,11 @@ export const NominatorRewardsList = () => {
     ]
   }, [selectedChain, pagination, isLargeLaptop])
 
+  const orderBy = useMemo(
```

Comment:
This code, or very similar code, is repeated multiple times. Can it be extracted into a function?